### PR TITLE
🚧Perform various improvements

### LINKED ISSUE #109: 🚧Perform various improvements
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

Perform various improvements/updates with things such as ide settings/config, workflows, readme badges, and more.

### Acceptance Criteria

- [x] Various improvements implemented

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [X] Issue linked to the correct project _(if applicable)_.
- [X] Issue linked to the correct milestone _(if applicable)_.
- [X] Draft pull request created and linked to this issue _(only required with code changes)_.

### Issue Dependencies

_No response_

### Related Work

_No response_

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| CICD Changes        | `♻️cicd` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
### **Description:**

To allow this pull request to be merged, please make sure that the following items of this pull request are complete.

This pull request closes #109

✅The pull request head branch must be a feature branch with the syntax 'feature/<issue-num>-sync-testing'.  
✅The pull request base branch must be a 'main' or 'preview' branch..  
✅The pull request head branch contains a valid issue number.  
✅The pull request title matches the linked issue title exactly.  
✅The pull request assignees match the assignees of the issue.  
✅The pull request labels match the labels of the issue.  
✅The pull request organizational projects match the organizational projects of the issue.  
✅The pull requests milestone matches the milestone of the issue.  

---

### **Additional Info:**
1. Pull requests are automatically synced with the associated issue upon creation by one of the KinsonDigital workflow bots. 
2. The associated issue is the issue number that is embedded in the pull request head branch.
3. The list above will be automatically updated as the pull request's various settings match or do not match the associated issue.
4. To manually sync the pull request to the issue, create a comment with the `[run-sync]` command.
	> **Note** You must be an admin member of the organization to use this command.

---

- [x] Sync with the issue.  Use this to enable or disable this pull request from syncing with its associated issue.

### CODE REVIEW

**Comment by CalvinWilkinson** [comment]
[run-sync]

File: .github/workflows/prepare-release.yml [hunk 1]
```diff
@@ -21,36 +21,38 @@ on:
...
-          if ($projectName -eq "") {
+          
+          if ("${{ vars.PROJECT_NAME }}".Trim() -eq "") {
             Write-Host "::error::The project name input cannot be empty.";
             exit 1;
           }
-
-
+          
+
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/prepare-release.yml]: Remove white space

```suggestion
```

File: .github/workflows/prepare-release.yml [hunk 2]
```diff
@@ -21,36 +21,38 @@ on:
...
+          
+          
           if ($releaseType -eq "") {
             Write-Host "::error::The release type input cannot be empty.";
             exit 1;
           }
-
-
+          
+
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/prepare-release.yml]: Remove white space

```suggestion
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
 
-env:
-  runs-on: ubuntu-latest
-
-
 on:
   workflow_dispatch:
     inputs:
-      release-type:
+      release_type:
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/release.yml]: Rename workflow input:

```suggestion
      release-type:
```

File: .github/workflows/sync-bot.yml [hunk 4]
```diff
@@ -13,34 +13,37 @@ on:
...
           $issueNumber = "${{ github.event.issue.number }}";
 
-          Write-Host "::notice::Project Name: ${{ vars.PROJECT_NAME }}";
-          Write-Host "::notice::Issue: $issueNumber";
+          Write-Host "::notice:Project Name: ${{ vars.PROJECT_NAME }}";
+          Write-Host "::notice:Issue: $issueNumber";
 
           if ($manuallyExecuted -and $issueNumber -eq "0") {
-            Write-Host "::error::The issue or PR number must be a value greater than 0.";
+            Write-Host "::notice:The issue or PR number must be a value greater than 0.";
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/sync-bot.yml]: Change this message type from a `notice` to and `error`

```suggestion
            Write-Host "::error::The issue or PR number must be a value greater than 0.";
```

File: .github/workflows/sync-status-check.yml [hunk 5]
```diff
@@ -27,24 +27,26 @@ jobs:
...
-          Write-Host "::notice::Project Name: ${{ vars.PROJECT_NAME }}";
-          Write-Host "::notice::PR Number: $prNumber";
-          Write-Host "::notice::Event Type: pr";
+          Write-Host "::notice:Project Name: ${{ vars.PROJECT_NAME }}";
+          Write-Host "::notice:PR Number: $prNumber";
+          Write-Host "::notice:Event Type: pr";
 
           if ($manuallyExecuted -and $prNumber -eq "0") {
-            Write-Host "::error::The issue or PR number must be a value greater than 0.";
+            Write-Host "::notice:The issue or PR number must be a value greater than 0.";
```
Comment (KinsonDigitalAdmin) [inline .github/workflows/sync-status-check.yml]: Refactor the message type from `notice` to `error`.

```suggestion
            Write-Host "::error::The issue or PR number must be a value greater than 0.";
```

File: .github/workflows/unit-test-status-check.yml
Comment (KinsonDigitalAdmin) [inline .github/workflows/unit-test-status-check.yml:1]: Add empty line to the end of the file

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by KinsonDigitalAdmin — DISMISSED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Pull request merged**